tests/crates/enarx_exec_tests/src/lib.rs

```diff
@@ -8,6 +8,7 @@ macro_rules! musl_fsbase_fix {
         /// Set FSBASE
         ///
         /// Overwrite the only location in musl, which uses the `arch_prctl` syscall
+        #[cfg(target_os = "linux")]
```

Comment:
```suggestion
        #[cfg(all(test, target_arch = "x86_64",  target_vendor = "unknown", target_os = "linux", target_env = "musl"))]
```

Reply:
I don't think `test` is a.) necessary, b.) actually used, since this is a bindep